warp10/src/main/java/io/warp10/script/functions/SETTO.java

```diff
@@ -29,24 +29,26 @@
  * Convert the set on top of the stack into a list
  */
 public class SETTO extends NamedWarpScriptFunction implements WarpScriptStackFunction {
-  
+
   public SETTO(String name) {
     super(name);
   }
-  
+
   @Override
   public Object apply(WarpScriptStack stack) throws WarpScriptException {
     Object top = stack.pop();
-    
-    if (!(top instanceof Set)) {
-      throw new WarpScriptException(getName() + " expects a set on top of the stack.");
+
+    if (top instanceof Set) {
+      List<Object> list = new ArrayList<Object>();
+
+      list.addAll((Set) top);
+
+      stack.push(list);
+    } else if (top instanceof List) {
+      stack.push(top);
```

Comment:
If the list contains duplicates, they are not be removed.
This is perhaps a odd behavior for lists that pass through a SET function